benchmarks/filtsmooth.py

```diff
@@ -0,0 +1,160 @@
+"""Benchmarks for Gaussian filtering."""
+import numpy as np
+
+import probnum.filtsmooth as pnfs
+import probnum.statespace as pnss
+from probnum.randvars import Normal
+
+
+def load_pendulum():
+    """Set up pendulum problem"""
+
+    np.random.seed(12345)
+
+    delta_t = 0.0075
+    var = 0.32 ** 2
+    g = 9.81
+
+    def f(t, x):
+        x1, x2 = x
+        y1 = x1 + x2 * delta_t
+        y2 = x2 - g * np.sin(x1) * delta_t
+        return np.array([y1, y2])
+
+    def df(t, x):
+        x1, x2 = x
+        y1 = [1, delta_t]
+        y2 = [-g * np.cos(x1) * delta_t, 1]
+        return np.array([y1, y2])
+
+    def h(t, x):
+        x1, x2 = x
+        return np.array([np.sin(x1)])
+
+    def dh(t, x):
+        x1, x2 = x
+        return np.array([[np.cos(x1), 0.0]])
+
+    q = 1.0 * (
+        np.diag(np.array([delta_t ** 3 / 3, delta_t]))
+        + np.diag(np.array([delta_t ** 2 / 2]), 1)
+        + np.diag(np.array([delta_t ** 2 / 2]), -1)
+    )
+    r = var * np.eye(1)
+    initmean = np.ones(2)
+    initcov = var * np.eye(2)
+    dynamod = pnss.DiscreteGaussian(2, 2, f, lambda t: q, df)
+    measmod = pnss.DiscreteGaussian(2, 1, h, lambda t: r, dh)
+    initrv = Normal(initmean, initcov)
+    return dynamod, measmod, initrv, {"dt": delta_t, "tmax": 4}
```

Comment:
why is this copied here and not referenced?

Reply:
Because it is not part of our problem zoo. Shall I use a different problem?

Reply:
so as soon as we move this to the zoo (which would happen soon, right?), it would be referenced here?

Reply:
I think so, yes. So far, we only use the problem in the `test`s. Do you want me to add this to the problem zoo within this PR? Would be no problem 👍

Reply:
we dont have a filtsmooth zoo, right? In this case it would be great if you took care of it in a different PR? What do you think?

Reply:
Yep, sure, will do :)